### PR TITLE
Publish Aerogear Digger Changes

### PR DESCRIPTION
**Pull request by laurafitzgerald**
**JIRA**
https://issues.jboss.org/browse/AGDIGGER-198

**What**
Merging all changes on master to production to publish the site following AeroGear Digger changes

**Motivation**
Now that we have a set way to add new changes (see README.md) lets get all outstanding changes from master merged to production. We should then follow the new process so that staging and publishing happens on a smaller scale so is easier to consume and review.

**Verification**
Master is published to https://staging.aerogear.org for verification

@aliok @matzew @finp @pmacko1

### CODE REVIEW

**Comment by pmacko1** [comment]
@laurafitzgerald let me go through the docs once again to be sure

File: .gitignore [hunk 1]
```diff
@@ -11,4 +11,5 @@ bin/
 *~
```
Comment (pmacko1) [inline .gitignore]: let me go through the docs, i will "approve" then

**Comment by pmacko1** [comment]
we should update https://github.com/aerogear/aerogear.org/blob/master/digger/index.html .  i don't see any descriptive value in it

**Comment by laurafitzgerald** [comment]
@pmacko1 lets get a JIRA created for that. We should still merge the changes and then follow the steps as listed in the description for any new changes.

**Comment by pmacko1** [comment]
@laurafitzgerald https://issues.jboss.org/browse/RHMAP-18954

**Comment by pmacko1** [comment]
@laurafitzgerald my review is more style/readability/clarity wise, i think the info presented is 99% the same, approved

**Comment by laurafitzgerald** [comment]
@pmacko1 what did you mean when you said `running steps individually with steps is not present here`

**Comment by pmacko1** [comment]
@laurafitzgerald paragraph describing running playbook steps(plays?  ...?) individually with tags is not present here (is in newer pr's)

**Comment by laurafitzgerald** [comment]
@pmacko1 okay. if it is added in a later pr then that one should be merged to master and then merged to production branch.

**Pull request merged**